<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>КалькуляторQtDesigner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>282</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Решение</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Коэффициент а</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="get_a">
         <property name="contextMenuPolicy">
          <enum>Qt::PreventContextMenu</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>8</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="placeholderText">
          <string>Коэффициент а</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Коэффициент b</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="get_b">
         <property name="contextMenuPolicy">
          <enum>Qt::PreventContextMenu</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>8</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="placeholderText">
          <string>Коэффициент b</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>160</width>
        <height>84</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Вид выражения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_sloj">
         <property name="text">
          <string>Сложение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_vich">
         <property name="text">
          <string>Вычитание</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_ymnoj">
         <property name="text">
          <string>Умножение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_del">
         <property name="text">
          <string>Деление</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>61</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="vichisl">
         <property name="text">
          <string>Вычислить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ochist">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sohran">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="pokaz">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>35</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>История</string>
     </attribute>
     <widget class="QTextEdit" name="show_history_vich">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>231</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>230</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="Open"/>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_program"/>
    <addaction name="about_author"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Стиль вывода</string>
    </property>
    <addaction name="Styles"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="Open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="about_program">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="about_author">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="actionsfdg">
   <property name="text">
    <string>sfdg</string>
   </property>
  </action>
  <action name="Styles">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>get_a</sender>
   <signal>textChanged(QString)</signal>
   <receiver>pokaz</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ochist</sender>
   <signal>pressed()</signal>
   <receiver>get_a</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ochist</sender>
   <signal>pressed()</signal>
   <receiver>get_b</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
